handler/src/main/java/io/netty/handler/ssl/JdkAlpnSslEngine.java

```diff
@@ -39,7 +40,8 @@ static boolean isAvailable() {
     }
 
     private static void updateAvailability() {
-        if (available) {
+        // TODO: Add support for ALPN when using Java9 and still be able to configure it the Netty way.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1135)